talk/morelanguage/exceptions.tex

```diff
@@ -46,32 +46,60 @@
   \begin{block}{Throwing exceptions}
     \begin{itemize}
       \item objects of any type can be thrown (even e.g.\ \cppinline{int})
+    \end{itemize}
+    \begin{goodpractice}{Throwing exceptions}
       \begin{itemize}
-        \item prefer standard exception classes
+        \item prefer throwing standard exception classes
+        \item throw objects by value
       \end{itemize}
-      \item throw objects by value
-    \end{itemize}
+    \end{goodpractice}
   \end{block}
```

Comment:
```suggestion
```

Reply:
and maybe add a `\small` so that the code fits better

Reply:
done